backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -330,36 +135,10 @@ private void createAdminUser(String firstName, String lastName, String email, St
     _aus.createUser(email, adminName, externalId, Role.ADMIN);
   }
 
-  // This is for temporary compatibility so the request can contain either the type label (old way)
-  // or the type name (new way).  Once the request is updated, then we only need to validate the
-  // type with `parseOrganizationType`
-  private String getTypeFromLabelOrValue(String labelOrValue) {
-    try {
-      return Translators.parseOrganizationTypeFromName(labelOrValue);
```

Comment:
I think we can also get rid of `parseOrganizationTypeFromName` now if we want to.